tests/main/lxd/task.yaml

```diff
@@ -0,0 +1,34 @@
+summary: Ensure that lxd works
+
+execute: |
+    wait_for_lxd(){
+        while ! printf "GET / HTTP/1.0\n\n" | nc -U /var/snap/lxd/common/lxd/unix.socket | MATCH "200 OK"; do sleep 1; done
+    }
+
+    echo "Install lxd"
+    snap install lxd
+
+    echo "Create a trivial container using the lxd snap"
+    wait_for_lxd
+    lxd init --auto
+    lxd.lxc launch ubuntu:16.04 my-ubuntu
```

Comment:
@mvo5 sorry for the very late response, IMO it's tricky because, given that we use lxd from the snap, if we initialize the linode image with it and put a container image in place, when the suite is initialized all the previous state would be whipped. 

One possible solution could be to make the container image available at some well-known location in the linode image and, after the lxd snap is installed during the test, move the container image files to /var/snap/lxd/common/lxd/images, so that we don't need to download it during the test, do you think that would be a good approach?